pilot/pkg/xds/eds.go

```diff
@@ -434,17 +434,6 @@ func canSendPartialFullPushes(req *model.PushRequest) bool {
 	return true
 }
 
-func canUseOldEdsCache(req *model.PushRequest) bool {
```

Comment:
Is this removal intentional? How are we handling PA changes now?

Reply:
Yes, i have moved this handling to 
```
func (x XdsCacheImpl) Clear(s sets.Set[ConfigKey]) {
	x.cds.Clear(s)
	// clear all EDS cache for PA change
	if HasConfigsOfKind(s, kind.PeerAuthentication) {
		x.eds.ClearAll()
	} else {
		x.eds.Clear(s)
	}
	x.rds.Clear(s)
	x.sds.Clear(s)
}
```

Reply:
Got it